docs/README.md

```diff
@@ -0,0 +1,18 @@
+# Documentation for CDH ansible playbooks and project infrastructure
+
+## Application-specific documentation
+- [Prodigy](applications/prodigy.md) 
+
+
+## General
+
+CDH ansibilized applications are deployed to virtual servers managed by PUL.
+
+Typically servers are running Ubuntu Linux, and we have two VMs for each environment (staging and production). For historical reasons and for brevity, our server names use `-test` for staging environments (e.g., for PPA the staging servers ase cdh-test-prosody1 and cdh-test-prosody2).
+
+We access PUL-managed servers using ssh and the `pulsys` (PUL Systems) account.
+Access is based on ssh keys associated with team member GitHub accounts; this is managed
+in [princeton_ansible variables](https://github.com/pulibrary/princeton_ansible/blob/main/group_vars/all/vars.yml#L115). If you need to update ssh keys on a particular server, you can use the 
+"Update pulsys user keys" template in Ansible Tower and limit to the appropriate servers.
```

Comment:
I'm not sure what this means. Where would I find this template?

Reply:
Let me explain here and then you help me figure out what we should add to our documentation. I'm also asking if it's ok to link directly to templates.

Here's a link to the template I mean in Ansible Tower: https://ansible-tower.princeton.edu/#/templates/job_template/17/details

When you "launch" the template/playbook, there is an input box for "limit" which is equivalent to the `--limit` command when running a playbook locally, i.e. only run on the host or host group I specify rather than all the hosts/host groups listed in the playbook

Reply:
Ah, so this is within the Ansible Tower application.

I think this is my first time navigating to [ansible-tower.princton.edu](https://ansible-tower.princeton.edu), but I can't access any Templates (within the Resources drop down). This is also true when trying to navitgate to the link you shared.

Perhaps this is a permissions issue?

Reply:
From the demo at standup, to add new keys to the CDH group, modify https://github.com/pulibrary/princeton_ansible/blob/main/group_vars/all/vars.yml. Add to documentation :D